src/nikhilMJun20/CharClassify.java

```diff
@@ -0,0 +1,39 @@
+/*Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+Input: 1rRpd3F9#K(E
+Output : 
+Total digit: 3
+Total Letters: 7
+Total Uppercase: 4
+Total Lowercase: 3
+Total Special char: 2
+*/
+
+package nikhilMJun20;
+
+public class CharClassify {
+	void findCharType(String str) {
+		int digitCnt=0, letterCnt=0, upperCnt=0, lowerCnt=0, specialCnt=0;
+		
+		for(int index=0; index<str.length(); index++) {
+			if(Character.isDigit(str.charAt(index))) {
+				digitCnt++;
+			}else if(Character.isLetter(str.charAt(index))) {
```

Comment:
From line number 21 to 29, you could write some better logic, think about it. Or will let you know. No need to check else if and if conditions and letterCnt this variable is also not needed.

Reply:
Hi Krishna, 
I have modified the code.
But, please do let me know if there's better logic than this. I will modify accordingly.
Thanks!